<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite name="com.makaan.Test.LoginTest" tests="7" failures="1" timestamp="11 May 2016 13:58:41 GMT" time="130.734" errors="0">
  <testcase name="ValidateResetPassword" time="22.298" classname="com.makaan.Test.LoginTest"/>
  <testcase name="ValidateSocialLogin" time="30.109" classname="com.makaan.Test.LoginTest"/>
  <testcase name="Logout" time="4.343" classname="com.makaan.Test.LoginTest"/>
  <testcase name="Signup" time="18.105" classname="com.makaan.Test.LoginTest"/>
  <testcase name="ValidateLoginButton" time="0.064" classname="com.makaan.Test.LoginTest"/>
  <testcase name="ValidateForgotPassword" time="46.441" classname="com.makaan.Test.LoginTest">
    <failure type="java.lang.AssertionError" message="not able to Verify Forgot password  form expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: not able to Verify Forgot password  form expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:496)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at com.makaan.Test.LoginTest.ValidateForgotPassword(LoginTest.java:57)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:821)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:773)
	at org.testng.TestRunner.run(TestRunner.java:623)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1185)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1110)
	at org.testng.TestNG.run(TestNG.java:1018)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:137)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:58)
]]>
    </failure>
  </testcase> <!-- ValidateForgotPassword -->
  <testcase name="ValidateMakaanLogin" time="9.374" classname="com.makaan.Test.LoginTest"/>
</testsuite> <!-- com.makaan.Test.LoginTest -->
